docs/aggregation-api.md

```diff
@@ -0,0 +1,14 @@
+API for the aggregated multiparty computation protocol 
```

Comment:
It's probably good to have at least one line of text before this, since Rustdoc will use it as a summary line. If the summary line is a heading, then the heading will be inserted into other places in the docs where a heading doesn't look great.

Reply:
👍